pkgs/record_use/lib/src/data_classes/arguments.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+
+class Arguments {
+  ConstArguments constArguments;
```

Comment:
Why are some of those fields (e.g. in `Annotation` final) and others like here not?

Reply:
They should all be final probably - let me check.